src/client/src/app/site/[site]/view-availability/week/page.tsx

```diff
@@ -1,25 +1,56 @@
 import NhsPage from '@components/nhs-page';
+import {
+  assertPermission,
+  fetchDailyAvailability,
+  fetchSite,
+} from '@services/appointmentsService';
 import dayjs from 'dayjs';
+import isoWeek from 'dayjs/plugin/isoWeek';
+import isSameOrAfter from 'dayjs/plugin/isSameOrAfter';
+import isSameOrBefore from 'dayjs/plugin/isSameOrBefore';
```

Comment:
It doesn't look like we're actually using these imports in this file? 
We're also import them and extending dayjs with them in another file (copied from this one? ;) )

I know it means more wrappers but could we move them into `timeService` so all the extensions are applied in one place. Just avoids this sort of thing and makes it easier to replace dayjs if we ever decide to

Reply:
Good spot, this is me removing the need for them but forgetting to remove the imports :)